internal/web/datasethandler.go

```diff
@@ -390,15 +398,48 @@ func (handler *datasetHandler) getChangesHandler(c echo.Context) error {
 			return echo.NewHTTPError(http.StatusBadRequest, server.SinceParseErr(err).Error())
 		}
 		preStream()
-		continuationToken, err := dataset.ProcessChangesRaw(sinceNum, l, false, func(jsonData []byte) error {
-			_, _ = c.Response().Write([]byte(","))
-			_, _ = c.Response().Write(jsonData)
-			return nil
-		})
-		if err != nil {
-			return echo.NewHTTPError(http.StatusInternalServerError, err.Error())
+		if reverse {
```

Comment:
This is not pretty, and should be changed in the future, but I understand why it has to be like this (for now).